lib/devices/ios/xcode.js

```diff
@@ -86,7 +88,11 @@ xcode.getVersion = function (cb) {
     if (!fs.existsSync(plistPath)) {
       return cb(new Error("Could not get Xcode version: " + plistPath + " does not exist on disk."), null);
     } else {
-      var cmd = "/usr/libexec/PlistBuddy -c 'Print CFBundleShortVersionString' " + plistPath;
+      try {
+        var cmd = "/usr/libexec/PlistBuddy -c 'Print CFBundleShortVersionString' " + escapeSpace(plistPath);
+      } catch(e) {
+        return cb(e, null);
```

Comment:
another style point: we do `cb(e)`, not `cb(e, null)`, making sure to just not pass any value at all into the callback in the failure case. (I see that someone did this two lines above, however we probably should have asked for that to be changed)